azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADAuthenticationFilter.java

```diff
@@ -99,21 +60,21 @@ protected void doFilterInternal(
 
                 final ServiceEndpoints serviceEndpoints =
                         serviceEndpointsProp.getServiceEndpoints(aadAuthFilterProp.getEnvironment());
+                final AzureADGraphClient client = new AzureADGraphClient(aadAuthFilterProp, serviceEndpointsProp);
 
                 if (principal == null || graphApiToken == null || graphApiToken.isEmpty()) {
                     principal = new UserPrincipal(idToken, serviceEndpoints);
-                    graphApiToken = acquireTokenForGraphApi(
-                            idToken, principal.getClaim().toString(), serviceEndpoints).getAccessToken();
+                    graphApiToken =
+                            client.acquireTokenForGraphApi(idToken, principal.getClaim().toString()).getAccessToken();
```

Comment:
about line wrap, how about a local variable to hold `principal.getClaim().toString()` and pass to acquire for avoiding line wrap ?

Reply:
Yes, I  think it's better and also better readability, will change.